### ISSUE TITLE
[Bug] Filenames are escaped in modification dialog, yet are rendered correctly.

### ISSUE THREAD

**Issue by RokeJulianLockhart** [damionx7/Zettel-Notes-Documentation#46]
As https://github.com/damionx7/Zettel-Notes-Documentation/issues/44#issuecomment-1908144820 states:

<blockQuote>

```.MD
	`{title: "Bitwarden Note Template",	id: "s4zw11"}`{.JSON5}.txt.MD
```

...which, notably, is converted by Zettel in the filename modification window to:

```.MD
__title_ _Bitwarden Note Template_,_id_ _s4zw11____.JSON5_.txt.MD
```

</blockQuote>

This means that although Zettel renders filenames with FS/OS-disallowed characters, the user cannot edit these via Zettel. It seems like a strange restriction.

**Comment by damionx7** [comment]
Please read the filename specification that zettel notes follows here 
https://help.dropbox.com/organize/file-names

basically following chars are not allowed in filename
```
/ (forward slash)
\ (backslash)
< (less than)
> (greater than)
: (colon)
" (double quote)
| (vertical bar or pipe)
? (question mark)
* (asterisk)
. (period)
```

**Comment by RokeJulianLockhart** [comment]
> https://github.com/damionx7/Zettel-Notes-Documentation/issues/46#issuecomment-1910770526

Indeed, @damionx7, I'm aware that although ext4 itself allows all but `null` and `/`, AOSP disallows a myriad of them (per https://stackoverflow.com/a/64021421/9731176). I'm not requesting that we somehow bypass that (an impossibility). I'm requesting that, like the rendered filename, the edit form also converts the characters which the user provides to their underlying escaped form.
